sdk/dapp-kit/src/hooks/useSuiClientQueries.ts

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import type { UseQueryResult } from '@tanstack/react-query';
+import { useQueries } from '@tanstack/react-query';
+
+import { useSuiClientContext } from './useSuiClient.js';
+import type { SuiRpcMethods, UseSuiClientQueryOptions } from './useSuiClientQuery.js';
+
+export function useSuiClientQueries<
+	T extends keyof SuiRpcMethods,
+	TData = SuiRpcMethods[T]['result'],
+>(
+	...args: undefined extends SuiRpcMethods[T]['params']
+		? [
+				method: T,
+				params?: SuiRpcMethods[T]['params'],
+				options?: UseSuiClientQueryOptions<T, TData>,
+		  ][]
+		: [
+				method: T,
+				params: SuiRpcMethods[T]['params'],
+				options?: UseSuiClientQueryOptions<T, TData>,
+		  ][]
```

Comment:
should we make these objects?

This looks like it will require all methods to be the same, I think this would be more useful if we could allow each method to be different